README.md

```diff
@@ -298,11 +299,77 @@ By specifying the `contractname`, Timber will know: which contract to interact w
 E.g. `"contractname": "MerkleTreeControllerSHA"`
 
 
+#### `treeId`
+
+The structure above assumes you have one tree per contract. If your application requires multiple trees per contract, you can specify a `treeId` (and height) for each. This tells Timber to create a separate db collection for each id.
+E.g.:
+```js
+// contracts to filter:
+contracts: {
+  // contract name:
+  MerkleTreeControllerSHAZVM: {
+    treeId: {
+      a: {
+        treeHeight: 16,
+        events: {
+          // filter for the following event names:
+          NewLeafA: {
+            // filter for these event parameters:
+            parameters: ['leafIndex', 'leafValue'],
+          },
+          NewLeavesA: {
+            // filter for these event parameters:
+            parameters: ['minLeafIndex', 'leafValues'],
+          },
+        },
+      },
+      b: {
+        treeHeight: 10,
+        events: {
+          // filter for the following event names:
+          NewLeafB: {
+            // filter for these event parameters:
+            parameters: ['leafIndex', 'leafValue'],
+          },
+          NewLeavesB: {
+            // filter for these event parameters:
+            parameters: ['minLeafIndex', 'leafValues'],
+          },
+        },
+      },
+    },
+  },
+},
+```
+**Note:** If the contract has multiple trees, all requests to the API must specify the `treeid` in the body of the request.
+
+E.g. `{ "treeid": "a" }`
+
+If you submit a request to a contract with multiple trees without specifying `treeid`, Timber will throw an error. This is because it does not know which tree to direct the request to.
+
+The event names must differ between trees so Timber knows which db collection to add the new leaves to. As long as the name of the event adding single leaves contains the term `Leaf`, and that adding multiple leaves contains the term `Leaves`, they can have any structure you like.
```

Comment:
Double-check whether this strict naming requirement is enforced by the code

Reply:
Changed to the first listed event to be the 'new leaf' (single) event, and the second to be 'new leaves' (multiple) event. Names can be anything!